docs/sso/twitter.md

```diff
@@ -0,0 +1,33 @@
+# How to Setup Twitter SSO
+
+1. Install the Social login app in your NextCloud (see README.md)
+2. Apply for becoming a Twitter App Developer
+    1. Go to https://developer.twitter.com/
+    2. Click "Apply" for a developer account
+    3. Fill out relevant information
+        * You'll need to provide a valid phone number and confirm it
+        * Under "The specifics" when it asks you what your intent is, explain it is for NextCloud and then uncheck each box appropriately. If you're just using this for NextCloud you can uncheck all boxes since this will just be for SSO, it won't be analyzing Twitter data or generating content.
+    4. Submit your application.
+    5. Confirm your email.
+    6. Wait for your application to be approved
+3. Create an application
+    1. Go to https://developer.twitter.com/
+    2. Click "Create an app"
+    3. Fill out relevant information:
+        * Website URL: https://example.com/
+        * Enable Sign In with Twitter: Check this box
+        * Callback URLs: https://example.com/apps/sociallogin/oauth/twitter
```

Comment:
On some server configurations url may be https://example.com/index.php/apps/sociallogin/oauth/twitter

Reply:
I'll add a generic statement explaining these instructions assume "https://example.com/" is the base URL.